roles/core/pxe_stack/templates/equipment_profile.ipxe.j2

```diff
@@ -8,20 +8,23 @@ echo |
 
 # Ansible variables
 set eq-equipment-profile {{ item | replace('equipment_','') | trim }}
-set eq-architecture {{ hostvars[groups[item][0]]['equipment_profile']['hardware']['cpu']['architecture'] }}
+set eq-architecture ${arch}
 set eq-distribution {{ hostvars[groups[item][0]]['equipment_profile']['operating_system']['distribution']|lower }}
 set eq-distribution-major-version {{ hostvars[groups[item][0]]['equipment_profile']['operating_system']['distribution_major_version'] }}
 set eq-distribution-version {{ hostvars[groups[item][0]]['equipment_profile']['operating_system']['distribution_version'] | default('${eq-distribution-major-version}') }}
 set eq-console {{ hostvars[groups[item][0]]['equipment_profile']['console'] }}
 set eq-kernel-parameters {{ hostvars[groups[item][0]]['equipment_profile']['kernel_parameters'] }}
-set eq-repositories-releasever {{ hostvars[groups[item][0]]['equipment_profile']['operating_system']['repositories_releasever'] | default('${eq-distribution-major-version}') }}
+set eq-repositories-releasever {{ hostvars[groups[item][0]]['equipment_profile']['operating_system']['distribution_version'] | default('${eq-distribution-major-version}') }}
```

Comment:
This variable was useful when we were able to customize the releasever. Now, it is redundant with `eq-distribution-version` line 14.
In an attempt to simplify the code, I would remove this line and replace any use of `eq-repositories-releasever` with `eq-distribution-version`.

Reply:
Done